cross/cuda-cross.txt

```diff
@@ -0,0 +1,16 @@
+[binaries]
+c = ['/usr/bin/aarch64-linux-gnu-gcc']
+cpp = ['/usr/bin/aarch64-linux-gnu-g++']
+cuda = ['/usr/local/cuda/bin/nvcc', '-ccbin', '/usr/bin/aarch64-linux-gnu-gcc']
+ar = '/usr/bin/aarch64-linux-gnu-ar'
+strip = '/usr/bin/aarch64-linux-gnu-strip'
+ld = '/usr/bin/aarch64-linux-gnu-ld'
+
+[host_machine]
+system = 'linux'
+cpu_family = 'aarch64'
+cpu = 'aarch64'
+endian = 'little'
+
+[properties]
+cuda_link_args = ['-lstdc++']
```

Comment:
```
DEPRECATION: cuda_link_args in the [properties] section of the machine file is deprecated, use the [built-in options] section.
```

Reply:
It seemed to me that `cuda_link_args` is not supported by `[built-in option]`

And indeed, this works:

```ini
[properties]
cuda_link_args = ['-lstdc++']
```

But This doesn't:

```ini
[built-in option]
cuda_link_args = ['-lstdc++']
```

From the [compiler-options doc](https://mesonbuild.com/Builtin-options.html#compiler-options), there's only `c_link_args` & `cpp_link_args`.

This looks like a spurious DEPRECATION notice.

Reply:
Yup. As discussed on IRC, if this is a spurious deprecation notice then we obviously cannot change here, but we should fix it in a separate PR to stop showing spurious, untruthful recommendations.

Reply:
Created https://github.com/mesonbuild/meson/issues/14653
@eli-schwartz What is missing for this to be merged?